app/javascript/components/table/Row.vue

```diff
@@ -79,11 +84,15 @@
 
     computed: {
       hasMetadata () {
-        return this.item.metadata.length !== 0
+        return this.item.metadata && this.item.metadata.length !== 0
```

Comment:
To check if the array is not null, you should be able to use the following shortcut: `this.item.metadata && this.item.metadata.length`. 
If `length` returns 0 then you get false, otherwise you get true.